### PR TITLE
Added TICKER support for Huobi

### PR DESCRIPTION
**Pull request by DavideColdebella**
Added field to websocket channels, added "_ticker" method and updated the message handler

- [kind of] - Tested: seems to work. There are no errors when I run it and the messages are formatted in the same way as for the other exchanges. Not sure how to test further.
- [ ] - Changelog updated
- [ ] - Tests run and pass
- [ ] - Flake8 run and all errors/warnings resolved
- [ ] - Contributors file updated (optional)

It's my first time creating a pull request.

### CODE REVIEW

File: cryptofeed/exchanges/huobi.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ class Huobi(Feed):
         L2_BOOK: 'depth.step0',
         TRADES: 'trade.detail',
         CANDLES: 'kline',
+        TICKER = 'ticker'
```
Comment (D-Stacks) [inline cryptofeed/exchanges/huobi.py:35]: probably should be : `TICKER: 'ticker'`

Comment (DavideColdebellaMelanion) [reply cryptofeed/exchanges/huobi.py:35]: You are totally right, somehow it slipped me

**Review by DavideColdebellaMelanion — APPROVED** [review]

File: cryptofeed/exchanges/huobi.py (hunk 1 above)
Comment (DavideColdebella) [reply cryptofeed/exchanges/huobi.py]: You're totally right, somehow I missed it

File: cryptofeed/exchanges/huobi.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ class Huobi(Feed):
         L2_BOOK: 'depth.step0',
         TRADES: 'trade.detail',
         CANDLES: 'kline',
+        TICKER: 'ticker'
```
Comment (bmoscon) [inline cryptofeed/exchanges/huobi.py:35]: the tests fail because this was not imported. Are you sure you tested this code?

**Review by bmoscon — CHANGES_REQUESTED** [review]

File: cryptofeed/exchanges/huobi.py (hunk 2 above)
Comment (DavideColdebella) [reply cryptofeed/exchanges/huobi.py:35]: I tested it in a notebook where I was probably importing the 'TICKER' and 'Ticker' in some other cell, and this hid the issue. My bad, sorry.

I made a minimal code snipped and tested the ticker method again, I hope I haven't missed any issue this time.

**Pull request merged**